app/models/physical_storage.rb

```diff
@@ -9,6 +9,8 @@ class PhysicalStorage < ApplicationRecord
   has_one :asset_detail, :as => :resource, :dependent => :destroy, :inverse_of => false
   has_many :guest_devices, :through => :hardware
 
+  has_many :physical_disks, :dependent => :destroy, :inverse_of => false
```

Comment:
@EsdrasVP why not specify `:inverse_of => :physical_storage` ?

Reply:
@agrare Changed here, I asked @CharlleDaniel about what exactly happens with a `:inverse_of` and he clarified me.